doc/Makefile.am

```diff
@@ -369,7 +369,8 @@ AM_CFLAGS = \
 	$(CODE_COVERAGE_CFLAGS)
```

Comment:
commit message for c34a429065514815a0b2a2974e1535350993048f

s/system-install/system-installed/
s/wil/will/